docs/development/shareable-configs.md

```diff
@@ -50,12 +50,40 @@ If you use a non-scoped config, you must use a preset name!
+| GitHub with preset name with a tag | `github>abc/foo:xyz#1.5.4` | `xyz`     | `https://github.com/abc/foo` | `xyz.json`     | `1.5.4`        |
+
+#### GitLab
+
+| name                               | example use                | preset    | resolves as                  | filename       | Git tag        |
+| ---------------------------------- | -------------------------- | --------- | ---------------------------- | -------------- | -------------- |
+| GitLab default                     | `gitlab>abc/foo`           | `default` | `https://gitlab.com/abc/foo` | `default.json` | Default branch |
+| GitLab with preset name            | `gitlab>abc/foo:xyz`       | `xyz`     | `https://gitlab.com/abc/foo` | `xyz.json`     | Default branch |
+| GitLab default with a tag          | `gitlab>abc/foo#1.5.4`     | `default` | `https://gitlab.com/abc/foo` | `default.json` | `1.5.4`        |
+| GitLab with preset name with a tag | `gitlab>abc/foo:xyz#1.5.4` | `xyz`     | `https://gitlab.com/abc/foo` | `xyz.json`     | `1.5.4`        |
+
+#### Gitea
+
+| name                              | example use               | preset    | resolves as                 | filename       | Git tag        |
+| --------------------------------- | ------------------------- | --------- | --------------------------- | -------------- | -------------- |
+| Gitea default                     | `gitea>abc/foo`           | `default` | `https://gitea.com/abc/foo` | `default.json` | Default branch |
+| Gitea with preset name            | `gitea>abc/foo:xyz`       | `xyz`     | `https://gitea.com/abc/foo` | `xyz.json`     | Default branch |
+| Gitea default with a tag          | `gitea>abc/foo#1.5.4`     | `default` | `https://gitea.com/abc/foo` | `default.json` | `1.5.4`        |
+| Gitea with preset name with a tag | `gitea>abc/foo:xyz#1.5.4` | `xyz`     | `https://gitea.com/abc/foo` | `xyz.json`     | `1.5.4`        |
+
+#### Self-hosted Git
+
+| name                              | example use                     | preset    | resolves as                          | filename        | Git tag        |
+| --------------------------------- | ------------------------------- | --------- | ------------------------------------ | --------------- | -------------- |
+| Local default                     | `local>abc/foo`                 | `default` | `https://github.company.com/abc/foo` | `default.json`  | Default branch |
+| Local with preset path            | `local>abc/foo//path/xyz`       | `default` | `https://github.company.com/abc/foo` | `path/xyz.json` | Default branch |
+| Local default with a tag          | `local>abc/foo#1.2.5`           | `default` | `https://github.company.com/abc/foo` | `default.json`  | `1.2.5.`       |
+| Local with preset path with a tag | `local>abc/foo//path/xyz#1.2.5` | `default` | `https://github.company.com/abc/foo` | `path/xyz.json` | `1.2.5.`       |
```

Comment:
```suggestion
| Local default with a tag          | `local>abc/foo#1.5.4`           | `default` | `https://github.company.com/abc/foo` | `default.json`  | `1.5.4.`       |
| Local with preset path with a tag | `local>abc/foo//path/xyz#1.5.4` | `default` | `https://github.company.com/abc/foo` | `path/xyz.json` | `1.5.4.`       |
```

Let's use the same version number here for consistency. 😉

Reply:
Oh I missed it :sweat: fixed :)